.github/workflows/ci.yml

```diff
@@ -1,142 +1,195 @@
-      
+      run: |
+        curl -fLOSs --retry 3 http://olivevideoeditor.org/deps/boost-win.7z
+        7z x boost-win.7z
+
     - name: Acquire Crashpad
-      run: curl -fLOSs --retry 3 http://olivevideoeditor.org/deps/crashpad-win.7z && 7z x crashpad-win.7z
+      run: |
+        curl -fLOSs --retry 3 http://olivevideoeditor.org/deps/crashpad-win.7z
+        7z x crashpad-win.7z
 
     - name: Configure CMake
       # Use a bash shell so we can use the same syntax for environment variable
       # access regardless of the host operating system
       shell: bash
-      working-directory: ${{runner.workspace}}/build
-      # Note the current convention is to use the -S and -B options here to specify source 
-      # and build directories, but this is only available with CMake 3.13 and higher.  
+      working-directory: ${{ runner.workspace }}/build
+      # Note the current convention is to use the -S and -B options here to specify source
+      # and build directories, but this is only available with CMake 3.13 and higher.
       # The CMake binaries on the Github Actions machines are (as of this writing) 3.12
-      run: PATH=$GITHUB_WORKSPACE:$PATH cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE -G "Ninja"
+      run: |
+        PATH=$GITHUB_WORKSPACE:$PATH \
+          cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -G "Ninja"
 
     - name: Build
-      working-directory: ${{runner.workspace}}/build
+      working-directory: ${{ runner.workspace }}/build
       shell: bash
-      # Execute the build.  You can specify a specific target with "--target <NAME>"
-      run: cmake --build . --config $BUILD_TYPE
+      # Execute the build. You can specify a specific target with "--target <NAME>"
+      run: |
+        cmake --build . --config ${{ matrix.build-type }}
 
     - name: Test
-      working-directory: ${{runner.workspace}}/build
+      working-directory: ${{ runner.workspace }}/build
       shell: bash
-      # Execute tests defined by the CMake configuration.  
+      # Execute tests defined by the CMake configuration.
       # See https://cmake.org/cmake/help/latest/manual/ctest.1.html for more detail
-      run: ctest -C $BUILD_TYPE
+      run: |
+        ctest -C ${{ matrix.build-type }}
 
     - name: Create Package
-      working-directory: ${{runner.workspace}}/build
+      working-directory: ${{ runner.workspace }}/build
       shell: cmd
-      run: mkdir olive-editor && copy app\olive-editor.exe olive-editor && copy app\olive-editor.pdb olive-editor && windeployqt olive-editor\olive-editor.exe && copy %GITHUB_WORKSPACE%\bin\*.dll olive-editor && copy %GITHUB_WORKSPACE%\out\Default\crashpad_handler.exe olive-editor
-    
+      run: |
+        mkdir olive-editor
+        copy app\olive-editor.exe olive-editor
+        copy app\olive-editor.pdb olive-editor
+        windeployqt olive-editor\olive-editor.exe
```

Comment:
Technically, we should also run deploy for olive-crashhandler.exe (though it shouldn't be necessary because olive-editor.exe uses a superset of libraries).

Reply:
Yeah I'm fairly sure `olive-crashhandler.exe` isn't used anymore (though jury's out on whether it'll be used again in the future)